.github/workflows/main.yml

```diff
@@ -4,19 +4,19 @@ on:
     branches:
       - main
   pull_request:
-    branches:
-      - "**"
 jobs:
   ci:
     name: CI
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Install Corepack
+        run: npm install -g corepack
       - uses: actions/setup-node@v4
         with:
-          node-version: 18.16.1
+          node-version-file: .nvmrc
       - name: Install
-        run: yarn install --immutable
```

Comment:
@lishaduck  I think it would be better to keep the --immutable option.

Reply:
It's redundant, but fair enough.

<img width="1011" alt="Screenshot 2025-05-20 at 9 37 19 AM" src="https://github.com/user-attachments/assets/92a751cd-6825-41a6-a117-f0e127674101" />